deanhaleem/mario-clone-ts
# Audio(assets)[#7]: add audio assets and play audio on creation

### Github Issue

#7

### Description

Adding audio assets and also playing a sound effect on game creation. Should hear the one-up sound effect when loading the game.

### Testing Instructions

- Load the game normally with `npm run dev`
- Notice the one-up sound effect being played
